frontends/bmd/src/app_mark_voter_card_invalid.test.tsx

```diff
@@ -70,7 +70,7 @@ describe('Mark Card Void when voter is idle too long', () => {
     // User action removes Idle Screen
     fireEvent.click(screen.getByText('Yes, I’m still voting.'));
     fireEvent.mouseDown(document);
-    await advanceTimersAndPromises();
+    await advanceTimersAndPromises(0.2);
```

Comment:
Why was this change needed?

Reply:
Oh yeah I meant to leave a PR comment explaining this. There's a `setTimeout` with a delay of 200ms that triggers the closing of the idle screen after user activity. 200ms happened to be the value of BMD's `GLOBALS.CARD_POLLING_INTERVAL` before I replaced it with the card polling interval from the auth hook (which is only 100ms). `advanceTimersAndPromises` defaults to advancing timers by the card polling interval, so this was coincidentally "just working" before.